lib/internal/test_runner/mock.js

```diff
@@ -202,6 +202,30 @@ class MockTracker {
     return mock;
   }
 
+  getter(
+    object,
+    methodName,
+    implementation = kDefaultFunction,
+    options = kEmptyObject
+  ) {
+    return this.method(object, methodName, implementation, {
+      getter: true,
+      ...options
```

Comment:
Can you swap these two lines so that `getter` cannot be set to a different value. Same comment below for `setter`.